src/date/format.js

```diff
@@ -207,6 +208,25 @@ return function( date, numberFormatters, properties ) {
 
 			// Zone
 			case "z":
+
+				// z...zzz: "{shortRegion}", eg. "PST" or "PDT".
+				// zzzz: "{regionName} {Standard Time}" or "{regionName} {Daylight Time}",
+				// eg. "Pacific Standard Time" or "Pacific Daylight Time".
+				if ( properties.tzId ) {
+					ret = dateTimezoneRegionFormat(
+						date,
+						properties.regionFormatStandard,
+						properties.regionFormatDaylight,
+						properties.standardTzName,
+						properties.daylightTzName,
+						properties.exemplarCity
+					);
+					if ( ret ) {
+						break;
+					}
```

Comment:
Yes, this will not be required with check  for `properties.standardTzName and properties.daylightTzName